src/test/kotlin/step3/racing/CarImplTest.kt

```diff
@@ -0,0 +1,25 @@
+package step3.racing
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package step3.racing.car
```